### ISSUE TITLE
Loading callback does not fire for resources

### ISSUE THREAD

**Issue by acerix** [fullcalendar/fullcalendar#5278]
If only resources are provided by JSON, `loading` does not fire:

https://codepen.io/acerix/pen/abOJXwW?editors=0011

Reported in https://fullcalendar.freshdesk.com/a/tickets/1522

**Comment by cjsjyh** [comment]
Similar issue for me when using fullcalendar in React

I couldn't find examples of implementing loading so this is how i called FullCalendar component:
(TestFunctions are event functions.)
```
<FullCalendar 
    slotDuration='00:30:00'
    businessHours={businessHours}
    timeZone='Asia/Seoul'
    ref={this.calendarRef}
    locale={koLocale} 
    defaultView="timeGridWeek" 
    plugins={[timeGridPlugin, listPlugin, interactionPlugin]}
    eventAllow={this.eventAllowance}
    eventLimit={true}
    eventDurationEditable={true}
    editable={true}
    droppable={true}
    buttonIcons={true}
    eventSources={[this.testFunction, this.testFunction2]}
    eventClick={this.eventClick}
    eventDrop={this.eventUpdateTime}
    eventResize={this.eventUpdateTime}
    drop={this.externalEventDrop}
    select={this.select}
    allDaySlot={false}
    selectable={true}
    loading={this.loadingFunction}
    header={{
        left: "prev,next today",
        center: "title",
        right: "timeGridDay,timeGridWeek,listMonth"
    }}
/>
```

my loadingFunction:
```
    loadingFunction = (isLoading) => {
        console.log("Is Loading " + isLoading)
        this.isLoading = isLoading
    }
```

Output:
```
TestFunction1 Called
TestFunction2 Called
TestFunction1 Done!
Is Loading true
TestFunction2 Done!
```

As you can see, `loadingFunction` gets called way after event fetching starts.
On top of that, it doesn't get fired when the fetching is done

Am I doing anything wrong?

**Comment by arshaw** [comment]
fixed in v4.4.1
https://codepen.io/arshaw/pen/OJyeJVW?editors=0011

only resource-common needs to be updated to fix this bug, but might as well upgrade all possible fullcalendar packages (`npm upgrade`)
